runtime/litentry/src/lib.rs

```diff
@@ -148,7 +148,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	impl_name: create_runtime_str!("litentry-parachain"),
 	authoring_version: 1,
 	// same versioning-mechanism as polkadot: use last digit for minor updates
-	spec_version: 9185,
+	spec_version: 9186,
```

Comment:
Actually we don't need to bump the version - I just bumped it 3 days ago
But small problem